translate/src/translate.js

```diff
@@ -0,0 +1,56 @@
+// jshint esversion: 9
+
+const translator = 'https://translate.googleapis.com/translate_a/single';
+/**
+ * @description null
+ * @param {ParamsType} params list of command parameters
+ * @param {?string} commandText text message
+ * @param {!object} [secrets = {}] list of secrets
+ * @return {Promise<SlackBodyType>} Response body
+ */
+async function _command(params, commandText, secrets = {}) {
+
+  const axios = require('axios');
+  const sourceLang = 'auto';
+  let targetLang = params.language;
+  let translatedText = '';
+
+  if (!params.language) { targetLang = 'en'; }
+  if (!params.text) {
```

Comment:
This isn't required since commander complains about required params if they're not passed.